upup/pkg/fi/cloudup/bootstrapchannelbuilder_test.go

```diff
@@ -50,6 +51,20 @@ func TestBootstrapChannelBuilder_BuildTasks(t *testing.T) {
 	runChannelBuilderTest(t, "awsiamauthenticator", []string{"authentication.aws-k8s-1.12"})
 }
 
+func TestBootstrapChannelBuilder_JWKS(t *testing.T) {
+	h := testutils.NewIntegrationTestHarness(t)
+	defer h.Close()
+
+	h.SetupMockAWS()
+
+	featureflag.ParseFlags("+PublicJWKS")
+	unsetFeaureFlag := func() {
+		featureflag.ParseFlags("-PublicJWKS")
+	}
+	defer unsetFeaureFlag()
+	runChannelBuilderTest(t, "jwks", []string{"dns-controller.addons.k8s.io-k8s-1.12", "kops-controller.addons.k8s.io-k8s-1.16", "anonymous-access.addons.k8s.io-k8s-1.16"})
```

Comment:
```suggestion
	runChannelBuilderTest(t, "public-jwks", []string{"dns-controller.addons.k8s.io-k8s-1.12", "kops-controller.addons.k8s.io-k8s-1.16", "anonymous-access.addons.k8s.io-k8s-1.16"})
```

(with corresponding directory rename)

Reply:
Done!